packages/did/src/Did.chain.ts

```diff
@@ -284,9 +285,9 @@ export function serviceToChain(service: NewService): ChainDidService {
  * @returns The DID service.
  */
 export function serviceFromChain(
-  encoded: Option<DidServiceEndpointsDidEndpoint>
-): Service {
-  const { id, serviceTypes, urls } = encoded.unwrap()
```

Comment:
I took it to be a faulty implementation, because the did.query call has no optional service entries. But I now realise that if you query a service it would be an option...
I think it's ridiculous though to have this function duplicated in Did.rpc.ts with the only difference being that one expects an Option and the other doesn't. I'd suggest that we either change the function here to accept both option-wrapped and plain service types, or use this function in Did.rpc.ts. as-is by wrapping each service in an option before passing it in here.

Reply:
Not saying I like the way it is, I was just pointing it out, as it would be inconsistent with everywhere else. If we can reduce code duplication by wrapping it in an `Option`, then be it, I'd say.

Reply:
how do you like 2a45a708528fe022e60fcdd8722c78ddc697060a ?

Reply:
I personally prefer keeping the old signature for consistency and wrapping in `Option` where needed.

Reply:
we can also do that